pkgs/by-name/ba/backlight-control-rs/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  coreutils,
+  lib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "backlight_control_rs";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "DOD-101";
+    repo = pname;
```

Comment:
```suggestion
    repo = "backlight_control_rs";
```

Reply:
Why do this rather than using the pname variable?